Add scroll_container formspec element (redo) (#9101)
minetest/minetest: doc/lua_api.txt

* End of a container, following elements are no longer relative to this
  container.

### `scroll_container[<X>,<Y>;<W>,<H>;<scrollbar name>;<direction>;<scroll factor>]`

> * Start of a scroll_container block, all elements in a scroll_container block will

Reviewer: ```suggestion
* Start of a scroll_container block. All contained elements will ...
```

Reviewer 2: Done.

>   * take the block's coordinate as position origin,

Reviewer: ```suggestion
  * take the scroll container coordinate as position origin
```

Reviewer 2: Why should the comma be removed?

Reviewer: No particular reason. Probably happened by re-writing the line.

Reviewer 2: I'll mark this as resolved.

* End of a container, following elements are no longer relative to this
  container.

> ### `scroll_container[<X>,<Y>;<W>,<H>;<scrollbar name>;<direction>;<scroll factor>]`

Reviewer 3: call it `<orientation>`

* Start of a scroll_container block. All contained elements will ...
  * take the scroll_container coordinate as position origin,
  * be additionally moved by the current value of the scrollbar with the name
    `scrollbar name` times `scroll factor` in the direction `direction` and
  * be clipped to the rectangle defined by `X`, `Y`, `W` and `H`.

> * Possible values for `direction` are `down` and `right`.

Reviewer 3: ```suggestion
* Possible values for `orientation` are `vertical` and `horizontal`.
```

* End of a container, following elements are no longer relative to this
  container.

### `scroll_container[<X>,<Y>;<W>,<H>;<scrollbar name>;<orientation>;<scroll factor>]`

* Start of a scroll_container block. All contained elements will ...
  * take the scroll_container coordinate as position origin,
  * be additionally moved by the current value of the scrollbar with the name
    `scrollbar name` times `scroll factor` along the orientation `orientation` and
  * be clipped to the rectangle defined by `X`, `Y`, `W` and `H`.

> * Possible values for `orientation` are `vertical` and `horizontal`.

Author: Done.
